src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -14,7 +14,7 @@
 @Execute(onWikia = "mediawiki119")
 public class InfoboxBuilderTests extends NewTestTemplate {
```

Comment:
add here `@Test(groups = "InfoboxBuilderTests)"`, so you don't need to add this group for all tests